data/scripts.js

```diff
@@ -140,7 +140,10 @@ exports.BattleScripts = {
 				this.add('-notarget');
 				return true;
 			}
-			if (targets.length > 1) move.spreadHit = true;
+			if (targets.length > 1) {
+				this.attrLastMove('[spread]');
+				move.spreadHit = true;
+			}
```

Comment:
So something like this instead?
````
let hitTargets = [];
for (let i = 0; i < targets.length; i++) {
    let hitResult = this.tryMoveHit(targets[i], pokemon, move, true);
    if (hitResult || hitResult === 0 || hitResult === undefined) {
        moveResult = true;
        hitTargets.push(targets[i]);
    }
    damage += hitResult || 0;
}
if (move.spreadHit) this.attrLastMove('[spread] ' + hitTargets.join(','));
````
(character separating each target probably shouldn't be `,` but the general idea?)